### DISCUSSION TITLE
Start of work with Git

### DISCUSSION THREAD

**Discussion by camper5510** [community/community#22638] [New to GitHub]
<p>Hey guys, im begin my work with GIT. I got many information from network, but not found full instruction how work with that. For begin i wanna install git to my website( i think it will master branch) and wanna create dev branch. How i can doit that? Maybe someone can shared link with full instruction for begginers))) thanks all in advance</p>

**Comment by Li-Silke** [comment]
<p>Hi <a class="mention" href="/camper5510">@camper5510</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Cheers!</p>

**Comment by mpboom** [comment, marked as answer]
<p>I suggest you first learn a bit about Git:</p>
<p><a href="https://git-scm.com/book/en/v1/Getting-Started" rel="nofollow noopener">https://git-scm.com/book/en/v1/Getting-Started</a></p>
<p>Then take a look at with what technologies you want to build your site:</p>
<ul>
<li>Web server: NGINX/Node.js ?</li>
<li>Technologies: HTML/CSS/JS ?</li>
<li>Frameworks: Angular/React/Vue</li>
</ul>
<p>(or something completely different like PHP / <a href="http://ASP.NET" rel="nofollow noopener">ASP.NET</a> etc)</p>
<p>Then take a look at how you are going to host your site:</p>
<ul>
<li>GH Pages: <a href="https://pages.github.com/" rel="nofollow noopener">https://pages.github.com/</a> - it’s free</li>
<li>Or a VPS: <a href="https://digitalocean.com/" rel="nofollow noopener">https://digitalocean.com/ </a> <a href="https://linode.com/" rel="nofollow noopener">https://linode.com/</a> etc. - about $5/month and you should be fine</li>
<li>Or a shared hosting account somewhere - just Google it - from about $3/month</li>
</ul>
<p>(and of course a domain name)</p>

**Comment by mchez808** [comment]
<p>Hi camper.  Did you mean you want to use Git to push commits to the repo that contains your website?  Also, it’s my understanding that the master branch is set up by default when you create a new repo.</p>
<p>I have yet to learn about additional branches, but my learning plan is to check out the <a href="https://github.com/apps/github-learning-lab/" rel="nofollow noopener">GitHub Learning Lab</a>.</p>
<p>Happy learning!</p>
<p>-Mark</p>
